src/useVega.ts

```diff
@@ -0,0 +1,66 @@
+import { ref } from 'vue-demi'
+import { EmbedOptions, VisualizationSpec } from 'vega-embed'
+import { useVegaEmbed } from './useVegaEmbed'
+import { useVegaUpdater } from './useVegaUpdaters'
+import {
+  MaybeRef,
+  PlainObject,
+  RefElement,
+  SignalListeners,
+  VegaView
+} from './utils'
+
+export type VegaConfig = {
```

Comment:
Vega has a config already. Maybe call this something else.

Reply:
Changed to `VueVegaConfig`